runtime/src/lib.rs

```diff
@@ -316,7 +325,11 @@ fn handle_command(drivers: &mut Drivers) -> CaliptraResult<MboxStatusE> {
         CommandId::GET_IDEV_CSR => Err(CaliptraError::RUNTIME_UNIMPLEMENTED_COMMAND),
         CommandId::GET_IDEV_INFO => IDevIdInfoCmd::execute(drivers),
         CommandId::GET_LDEV_CERT => Err(CaliptraError::RUNTIME_UNIMPLEMENTED_COMMAND),
+        CommandId::INCREMENT_PCR_RESET_COUNTER => {
+            IncrementPcrResetCounter::execute(drivers, cmd_bytes)
+        }
         CommandId::INVOKE_DPE => InvokeDpeCmd::execute(drivers, cmd_bytes),
+        CommandId::QUOTE_PCRS => get_pcr_quote(drivers, cmd_bytes),
```

Comment:
nit: Follow existing convention of <ReqStruct>::execute

Reply:
I was wondering, what that was about. Since there isn't any data associated with those handlers, wouldn't it be better to just use modules? If that's not to your liking, maybe a trait makes sense in this case?